egs/multi_en/s5/path.sh

```diff
@@ -4,8 +4,5 @@ export PATH=$PWD/utils/:$KALDI_ROOT/tools/openfst/bin:$PWD:$PATH:$KALDI_ROOT/too
 [ ! -f $KALDI_ROOT/tools/config/common_path.sh ] && echo >&2 "The standard file $KALDI_ROOT/tools/config/common_path.sh is not present -> Exit!" && exit 1
 . $KALDI_ROOT/tools/config/common_path.sh
 
-# Sequitur G2P executable
-sequitur=$KALDI_ROOT/tools/sequitur/g2p.py
-sequitur_path="$(dirname $sequitur)/lib/$PYTHON/site-packages"
-
 export LC_ALL=C
+export LD_LIBRARY_PATH=$KALDI_ROOT/tools/openfst-1.6.2/lib/
```

Comment:
I'm not sure if this is necessary (it shouldn't be).

Reply:
removed